python/pyspark/sql/functions.py

```diff
@@ -2124,6 +2124,21 @@ def size(col):
     return Column(sc._jvm.functions.size(_to_java_column(col)))
 
 
+@since(2.4)
+def cardinality(col):
```

Comment:
Could you also remove this? I think we just need to do it for SQL functions now. In the future, if this is requested by the community, we can reconsider it. Thanks!

Reply:
I see.